containers/EditFish.js

```diff
@@ -1,10 +1,82 @@
-import React, { Component } from "react";
-import { StyleSheet, Text, View, Button, FlatList } from "react-native";
-
-export default function EditFish({ navigation }) {
-  return (
-    <View>
-      <Text>Edit Fish</Text>
-    </View>
-  );
-};
+import React, { Component }  from 'react';
+import FishForm from '../components/FishForm';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { setCurrentFish, clearFishEntry } from '../actions'
+import { addUpdatedFish } from "../ApiCalls";
+import { connect } from "react-redux";
+
+class EditFish extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            selectedFish: null
+        }
+    }
+
+    componentDidMount = () => {
+        const selectedFish = this.props.currentFish.find(fish => fish.id == this.props.selectedFishId)
+        this.setState({selectedFish: selectedFish})
+        this.setCurrentFish(selectedFish)
```

Comment:
There is a bug where when a user edits a fish, the fish value of weight, length, and fly type is not loaded.  The functionality works, but the display of the proper information is not showing.  We can address this in a future update to fix these bugs.

Reply:
Agreed! I think there is something with numbers rendering on the edit page on mobile devices since this issue (#66) also exists for editing a fly (the amount doesn't appear). I think the fly drop-down may be a different issue, but we definitely will want to look into both!